apps/sendsignal.go

```diff
@@ -21,7 +23,17 @@ func keepUniqueContainersWithNames(containers []scalingo.Container, names []stri
 			}
 		}
 		if _, ok := containersToKill[name]; !ok {
-			io.Errorf("The name '%v' did not match any container.\n", name)
+			hasMatched := false
+			prefix := fmt.Sprintf("%s-", name)
+			for _, container := range containers {
+				if strings.HasPrefix(container.Label, prefix) {
+					containersToKill[container.Label] = container
+					hasMatched = true
+				}
+			}
```

Comment:
this is exactly the same loop as above, isn't it possible to refactor and extract it in a dedicated function?